app/services/open_library/response_adapter.rb

```diff
@@ -0,0 +1,16 @@
+module OpenLibrary
+  class ResponseAdapter
+    def self.book_response(response_body, isbn)
+      raise Errors::BookNotFound if
```

Comment:
Not finding a book should be something that we should expect, raising an exception seems way too much.
How about adding a method to this class `success?` with this logic, so in the controller you could check it and respond adequately to the user depending on the case?